themes/finna2/templates/RecordDriver/DefaultRecord/data-allSubjectHeadingsExtended.phtml

```diff
@@ -14,15 +14,16 @@
         $first = $i++ === 0;
         $authId = $first && isset($field['id']) ? $field['id'] : null;
         $authType = $field['authType'] ?? null;
+        $detail = $first ? ($field['detail'] ?? null) : null;
       ?>
       <?= $first ? '' : ' &#8594; '?>
       <?php $subject = trim($subject . ' ' . $subfield); ?>
       <?php if ($authType && $authId): ?>
-        <?=$this->record($this->driver)->getAuthorityLinkElement('subject', $subfield, ['name' => $subject, 'id' => $authId], ['authorityType' => $authType, 'class' => ['backlink'], 'title' => $subfield])?>
+        <?=$this->record($this->driver)->getAuthorityLinkElement('subject', $subfield, ['name' => $subject, 'id' => $authId, 'description' => $detail], ['authorityType' => $authType, 'class' => ['backlink'], 'title' => $subfield, 'description' => true])?>
       <?php else: ?>
         <a class="backlink" title="<?=$this->escapeHtmlAttr($subject)?>" href="<?=$this->record($this->driver)->getLink('subject', $subject)?>">
           <?=trim($this->escapeHtml($subfield))?>
-        </a>
+        </a><?= $detail ? " ({$this->escapeHtml($detail)})" : '' ?>
```

Comment:
turhat aaltosulut?

Reply:
Ilman aaltosulkuja sulut tulostuvat tuplina:
![image](https://user-images.githubusercontent.com/3889/120156334-90304780-c1fa-11eb-93cf-7c555dc592ad.png)

Ehkä escape sequencet konfliktoi tässä jotenkin:
https://www.php.net/manual/en/language.types.string.php#language.types.string.parsing

Reply:
Sori, luin huonosti. Mieluummin ehkä `<?= $detail ? ' ('  . $this->escapeHtml($detail) . ')' : '' ?>

Reply:
tai ihan perinteisesti:
```
<?php if ($detail): ?>
  (<?=$this->escapeHtml($detail)?>
<?php endif; ?>
```

Reply:
Muutin käyttämään iffiä.

Reply:
Nyt siinä ei ole enää väliä a-tagiin. Onko tarkoituksella ihan kiinni?

Reply:
Ei sattumalta, vaan koska se on html:ää. :) Mut ok.